BASE/src/ServerTelemetryChannel/Implementation/TransmissionPolicy/AuthenticationTransmissionPolicy.cs

```diff
@@ -0,0 +1,120 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation.TransmissionPolicy
+{
+    using System;
+    using System.Threading.Tasks;
+
+    using Microsoft.ApplicationInsights.Channel.Implementation;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation;
+
+    /// <summary>
+    /// This class defines how the SDK will behave when it receives Response Codes 
+    /// from the Ingestion Service related to Authentication (AAD) scenarios.
+    /// </summary>
+    /// <remarks>
+    /// This class is disabled by default and expected to be enabled only when AAD has been configured in the AI SDK.
```

Comment:
"in the AI SDK" - i'd avoid these terms, and be more specific, even though these are internal code..